src/open_inwoner/templates/pages/profile/me.html

```diff
@@ -106,6 +106,33 @@ <h2 class="h2 title" id="notifications">{% trans "Voorkeuren voor meldingen" %}
     <section class="profile-section profile-section__overview">
         <h2 class="h2 title" id="profile-overview">{% trans "Overzicht" %}</h2>
         <div class="profile-cards card-container card-container--columns-3">
+            {# Selected categories #}
+            {% if view.config.selected_categories %}
+                <a href="{% url 'profile:categories' %}" class="card card--compact card--stretch" id="profile-selected-categories">
+                    <div class="card__body">
+                        <div class="profile__link">
+                            <p class="h4"><span class="link link__text">{% trans "Mijn Interessegebieden" %}</span></p>
```

Comment:
This is the wrong class (and therefore wrong font-size) - it needs to be like the other cards here:
```<div class="card__body">
   <div class="profile__link">
      <h4 class="card__heading-4"><span class="link link__text">{% trans "Mijn Interessegebieden" %}</span></h4>```

Reply:
Fixed it